contents/docs/user-guides/funnels.md

```diff
@@ -4,74 +4,175 @@ sidebar: Docs
+
+If all the data is grouped in one bar (known as a bin - in histogram speak), you can increase the number of bins to get a more granular view of the conversion time distribution.
+
+![image](https://user-images.githubusercontent.com/85295485/134484797-d30a5aeb-c2fe-4b6b-96bb-ef21689699e1.png)
+
+This graph illustrates how common it is to go through these steps at different paces, if you graph is skewed towards the left, this means most people are going through this step very fast, if the graph is skewed towards the right this means that people are going through this step slowly.
+
+You may see other patterns in the graph, including a double hump where you have two peaks in your graph, this indicates that there are two distinct types of experience or user, where some are going through fast and others slower. The group which are taking more time are likely experiencing significantly more friction than the group who are going through fast.
+
+
+### Identify possible causes of failure and success
+
+Now you have a funnel and can see people are struggling on certain steps. You now want to work out why. We have a tool designed specifically for this use case - to start you’ll want to think of a hypothesis of what might be causing the issue. A common one is people often struggle to use a web based product on a mobile device since it’s not optimized fully.
+
+Click on the breakdown in the configuration pane and select OS (Operating System) this will break down the funnel by operating system.
+
+![image](https://user-images.githubusercontent.com/85295485/134480856-9389ab45-8c49-4150-8e9a-5e6529595e0c.png)
+
+You can instantly see that the conversion rate is lower on Linux, this gives us an indication that it could be a cause of drop-off in this funnel. To validate it, click through to the persons modal to watch a session recording (if enabled on your instance) or try the flow yourself on the mobile device.
+
+You can break down by a number of user and event properties (e.g. browser, country, …) to help quickly validate the initial hypothesis you may have.
+
+Automating the solution to highlight potential problems and hypotheses for you is something we’re actively working on so, take a look on our roadmap if you’re interested to learn more about what we’re building here.
+
+
+### Understand how your changes are improving your flow
+
+There are multiple ways to understand how your changes, fixes and new features are affecting your funnel success rate - the simplest and most common is to look at how the conversion rate changes over time.
+
+In the configuration pane, switch to historical trends, this will show you how your conversion rate has changed over time. If you have launched a new feature a few weeks ago and expect it to improve your conversion rate - you should be able to see it through a change in the trend on the graph. You may wish to change the granularity and time range to focus on what happened after specific changes.
```

Comment:
```suggestion
In the configuration pane, switch to historical trends, this will show you how your conversion rate has changed over time. If you launched a new feature a few weeks ago and expect it to improve your conversion rate then you should be able to see this through a change in the trend on the graph. You may wish to change the granularity and time range to focus on what happened after specific changes.
```